src/components/clusters/ClustersTable.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
-import { Table, TableHeader, TableBody, TableVariant, IRowData } from '@patternfly/react-table';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  IRowData,
+  sortable,
```

Comment:
this needs to be used from `src/components/ui/table/sortable`

Reply:
done